tox.ini

```diff
@@ -14,6 +15,8 @@ usedevelop = True
 passenv = *
 setenv =
     PYTHONDONTWRITEBYTECODE=1
+    unit: PYTEST_ADDOPTS=test/unit/ --cov={toxinidir}/molecule/ --no-cov-on-fail
```

Comment:
I am totally against defining the PYTEST_ADDOPTS variable which has a very different purpose: of allowing the *user* to customize/override the pytest execution at runtime. For example on my machine I do have this defined in shell rc file PYTEST_ADDOPTS='-s -x --no-cov' --- and works fine with hundreads of projects using tox.

Reply:
fixed